tutorials/platform/android/android_in_app_purchases.rst

```diff
@@ -1,17 +1,11 @@
-:article_outdated: True
-
 .. _doc_android_in_app_purchases:
 
 Android in-app purchases
 ========================
 
 Godot offers a first-party ``GodotGooglePlayBilling`` Android plugin compatible with Godot 3.2.2 and higher.
```

Comment:
Given that these are the 4.x docs, does it still make sense to reference "Godot 3.2.2 and higher" here?